roles/lib_utils/tasks/main.yml

```diff
@@ -8,4 +8,6 @@
   package:
     name: python-ruamel-yaml
```

Comment:
Yes, they should be uniform. Looking through the packages that are in Fedora and in our repo python2-ruamel-yaml seems to be the package that's common amoungst them, however is this the right package to install on a target host that's running Fedora 25? I'm not sure if ansible will use python2 or python3?

```
[root@ose3-master ~]# rpm -q --provides python2-ruamel-yaml
python-ruamel-yaml = 0.12.14-9.el7
python-ruamel.yaml = 0.12.14-9.el7
python2-ruamel-yaml = 0.12.14-9.el7
python2-ruamel-yaml(x86-64) = 0.12.14-9.el7

[root@ose3-master ~]# rpm -qp -provides https://kojipkgs.fedoraproject.org//packages/python-ruamel-yaml/0.13.13/3.fc25/x86_64/python2-ruamel-yaml-0.13.13-3.fc25.x86_64.rpm
python-ruamel.yaml = 0.13.13-3.fc25
python-ruamel.yaml(x86-64) = 0.13.13-3.fc25
python2-ruamel-yaml = 0.13.13-3.fc25
python2-ruamel-yaml(x86-64) = 0.13.13-3.fc25
python2.7dist(ruamel.yaml) = 0.13.13
python2dist(ruamel.yaml) = 0.13.13
```